<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CfgDetails</class>
 <widget class="QWidget" name="CfgDetails">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CfgDetails</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Config</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="nameLabel">
          <property name="text">
           <string>name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="nameEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="idLabel_2">
          <property name="text">
           <string>id</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="idEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="commentLabel">
          <property name="text">
           <string>comment</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="commentEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="deleteCfgButton">
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
